inst-support-eng/panda-pto
# add quarterly point seeding

- closes #55 

#### test plan:
1. find (or make) a user who has taken *more* than 45 points worth of pto request
1. find (or make) a user who has taken *less* than 45 points worth of pto request
1. locally add the current date to the quarters array (line 6 app/controllers/pages_controller.rb)
1. run `rake quarterly_seed`
1. verify the user from step 1's bank_value increased by 45 points
1. verify the user from step 2's bank_value updated to 180
